sdk/containerregistry/azure-containerregistry/azure/containerregistry/_models.py

```diff
@@ -211,53 +266,82 @@ def _to_generated(self) -> RepositoryWriteableProperties:
             can_list=self.can_list,
         )
 
-    def __getattr__(self, name):
+    def __getattr__(self, name: str) -> datetime:
         if name == "last_udpated_on":
             warnings.warn(
-                "The property name with a typo called 'last_udpated_on' has been deprecated and will be retired in future versions",  # pylint: disable=line-too-long
+                "The property name with a typo called 'last_udpated_on' has been deprecated and will be retired \
+                in future versions",
                 DeprecationWarning,
             )
-            return self.last_updated_on
-        return super().__getattr__(self, name)  # pylint: disable=no-member
+        return self.last_updated_on
```

Comment:
We can't do this or every single attribute will become `last_updated_on`

Reply:
You're right. Since this class doesn't have a parent class, we can't return `super().__getattr__()`. After I tested, I found only unknown properties would call this function, e.g. obj.unknown. So I think we should return an `AttributeError` for other property names.